backend/Tasque.Core.Common/Entities/User.cs

```diff
@@ -15,6 +15,7 @@ public User()
         OwnedTasks = new List<Task>();
         ParticipatedTasks = new List<Task>();
         Roles = new List<UserProjectRole>();
+        SystemRoles = new List<UserOrganizationRole>();
```

Comment:
I don't think that `SystemRole` is a good title here

Reply:
It's already in dev, just removing backend warning